index.html

```diff
@@ -171,7 +171,7 @@ <h2>About BIDS</h2>
         <p>A good introduction to the BIDS standard can be found in the <a href="http://www.nature.com/articles/sdata201644">paper published in Nature Scientific Data</a>.</p>
         <p>Look through some of our and collaborators <a href="https://osf.io/yn93h/">presentations on BIDS</a>.</p>
```

Comment:
This sounds confusing: Who is BIDS if not the BIDS collaborators/community? --> I suggest to drop the separation

```suggestion
        <p>Look through some of our <a href="https://osf.io/yn93h/">presentations on BIDS</a>.</p>
```

Reply:
I think it is better to make it clear it is the community at least and giving credit

```suggestion
       <p>Look through some of the community's <a href="https://osf.io/yn93h/">presentations on BIDS</a>.</p>
```